src/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { MdHome } from 'react-icons/md';
+import styles from './BreadCrumb.module.scss';
+
+interface BreadcrumbItem {
+  items: {
+    title: string;
+    href: string;
+  }[];
+}
+
+export const BreadCrumb: React.FC<BreadcrumbItem> = ({ items }) => {
```

Comment:
細かいけど Breadcrumb で 1 語なので
```suggestion
export const Breadcrumb: React.FC<BreadcrumbItem> = ({ items }) => {
```
がよさそう